### PR TITLE
Replace time-picker d2l-input with d2l-text-input

### PR DESCRIPTION
**Pull request by awikkerink**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -4,7 +4,6 @@
   "main": "d2l-date-picker.html",
   "dependencies": {
     "polymer": "Polymer/polymer#^1.9.0",
-    "d2l-time-picker": "BrightspaceUI/time-picker#^0.3.0",
```
Comment (alogan519) [inline bower.json]: Looks like you need to add d2l-text-input

Comment (awikkerink) [inline bower.json]: It's not quite ready yet

Comment (awikkerink) [inline bower.json]: Done

**Comment by awikkerink** [comment]
ping @alogan519 @dlockhart

File: d2l-date-picker.html [hunk 2]
```diff
@@ -67,11 +67,11 @@
 		</custom-style>
 
 		<label for="input">{{label}}</label>
-		<vaadin-date-picker-light min="[[min]]" max="[[max]]" value="{{value}}" on-value-changed="handleValueChanged">
+		<vaadin-date-picker-light min="[[min]]" max="[[max]]" value="{{value}}" on-value-changed="handleValueChanged" attr-for-value="value">
 			<iron-a11y-keys target="[[_target]]" keys="enter" on-keys-pressed="_onEnter"></iron-a11y-keys>
 			<iron-a11y-keys target="[[_target]]" keys="up down" on-keys-pressed="_onUpDown"></iron-a11y-keys>
 			<iron-input>
-				<input id="input" on-tap="_handleTap" placeholder="{{placeholder}}" class="d2l-input" is="iron-input" type="text" on-focus="_handleFocus" aria-describedby="description"/>
+				<d2l-text-input id="input" on-tap="_handleTap" placeholder="{{placeholder}}" class="input" type="text" on-focus="_handleFocus" aria-describedby="description"></d2l-text-input>
```
Comment (dlockhart) [inline d2l-date-picker.html]: Is the `class="input"` still required? The existing `aria-describedby` seems... wrong.

Should it be `d2l-date-picker-description`?

Comment (awikkerink) [inline d2l-date-picker.html]: #20 fixes the describedby

`class="input"` is required for the Polymer 1.9 version of vaadin-date-picker

**Pull request merged**